packages/lib/modules/tokens/approvals/permit2/signPermit2Init.tsx

```diff
@@ -0,0 +1,37 @@
+import { ensureError } from '@repo/lib/shared/utils/errors'
+import { Address, Permit2, Permit2Helper, PublicWalletClient, InitPoolInputV3 } from '@balancer/sdk'
+
+type SignPermit2InitParams = {
+  sdkClient?: PublicWalletClient
+  account: Address
+  initPoolInput: InitPoolInputV3
+}
+
+export async function signPermit2Init(params: SignPermit2InitParams): Promise<Permit2 | undefined> {
```

Comment:
didn't we do permit2 before for other ops? if this is specific for creating pools we should probably change the name/filename to something more specific.

Reply:
`Init` is short for initialization which is what we call the first add liquidity operation to a pool that has been created but has no tokens yet. is a special operation that can only happen one time and must happen before regular add liquidity operations are allowed

i'm good with something like `signPermit2PoolInitialization` if you prefer?

Reply:
🙏 It was confusing for me (don't know if it is because I don't have the full vocabulary gasped yet)